src/process/data-provider.js

```diff
@@ -5,23 +5,41 @@ import {DateTime} from "luxon";
 
 const from = `${parseInt(Config.PROCESS_WINDOW) - 1} HOURS`;
 const to = `${Config.PROCESS_WINDOW} HOURS`;
+
 const enrichedDataQuery = `
-    select * from enriched_data
+    select *
+    from enriched_data
     where station = $1
-    and timestamp <= now() - interval '${from}'
-    and timestamp >= now() - interval  '${to}'
+      and timestamp <= now() - ($2 || ' hours'):: INTERVAL
+      and timestamp >= now() - ($3 || ' hours'):: INTERVAL
 `;
 
+const archiveEnrichedData =
+    `select *
```

Comment:
grawis (`) możesz przenieść do linii wyżej, a sam select * from tabela nie ma sensu rozbijać raczej, bo to nie jest jakieś kosmicznie długie